File: Gitesh2808/Java/simple-interest.java
// Date : 20 Jan 2021

// Java program to display the simple interest

            import java.io.*;
            class si
            {
              public static void main(String args[])
              {
                try 
                {
                  DataInputStream x=new DataInputStream(System.in);
                  int p, n, r;
                  float si;
                  System.out.print("\nEnter p, n, r : \n");
                  p = Integer.parseInt(x.readLine());
                  n = Integer.parseInt(x.readLine());
                  r = Integer.parseInt(x.readLine());
                  si = (p * n * r) / 100;
                  System.out.print("\nSimple Interest is : " + si);
                }
                catch(Exception e)
                {}
              }
            }

File: Gitesh2808/C/simple-interest.c
// Date : 21 Jan 2021

// C Program to calculate the simple interest



#include <stdio.h>
#include <stdlib.h>

int main()
{
    int p, n, r;
    float si;
    printf("Enter the values of p, n and r : \n");
    scanf("%d %d %d", &p, &n, &r);
    si = (p * n * r) / 100;
    printf("The simple interest is %f",si);
    return 0;
}


/*
Output :
    Enter the values of p, n and r : 
    21000
    3
    2
    
    The simple interest is 1260.000000
*/
